lambdas/services/virus_scan_result_service.py

```diff
@@ -0,0 +1,133 @@
+import json
+import os
+
+import requests
+from botocore.exceptions import ClientError
+from enums.lambda_error import LambdaError
+from enums.pds_ssm_parameters import SSMParameter
+from requests.models import HTTPError
+from services.base.ssm_service import SSMService
+from utils.audit_logging_setup import LoggingService
+from utils.lambda_exceptions import VirusScanResultException
+
+logger = LoggingService(__name__)
+
+FAIL_SCAN = "Virus scan result failed"
+
+
+class VirusScanResultService:
+    def __init__(self):
+        self.staging_s3_bucket_name = os.getenv("STAGING_STORE_BUCKET_NAME")
+        self.ssm_service = SSMService()
+        self.username = ""
+        self.password = ""
+        self.base_url = ""
+        self.access_token = ""
+
+    def prepare_request(self, file_ref):
+        try:
+            self.get_ssm_parameters_for_request_access_token()
```

Comment:
Would this fit better in the service __init__?

Reply:
I don't think so. As we wrap it with a try block here, and catch ClientError.